src/packages/dictionary/dictionary/workspace/views/editor/workspace-view-dictionary-editor.element.ts

```diff
@@ -62,12 +62,11 @@ export class UmbWorkspaceViewDictionaryEditorElement extends UmbLitElement {
 	render() {
 		return html`
 			<uui-box>
-				<p>Edit the different language versions for the dictionary item '<em>${this._dictionary?.name}</em>' below.</p>
-
+				${unsafeHTML(this.localize.term('dictionaryItem_description', this._dictionary?.name || 'unnamed'))}
```

Comment:
Do we have a translation for the word "unnamed" or how did this work in the current backoffice?

Reply:
I was unable to find any fitting candidate for this.

In the current backoffice, you have to enter a name before you can go to this page. It looks like the name is then "locked in" the translation until you press save.